### ISSUE TITLE
Use with Passport and/or Express

### ISSUE THREAD

**Issue by gendronb** [ericrglass/fastify-normalize-request-reply#1]
@ericrglass 

Would this plugin help solve any of these issues ?

https://github.com/fastify/fastify/issues/1350
https://github.com/fastify/fastify/issues/1503

**Comment by ericrglass** [comment]
Hi Bruno,

I have only tested the plugin with several of the express middleware.
Mostly I wanted to use the swagger-stats with fastify within nestjs, and
have only gotten a few others to work.

I have never tried passport,until you asked your question, so I did put
together a simple test project and see that passport is both middleware and
a prehandler/route handler. It is also dependent with the express
body-parser middleware, which is one that I could not get to work.

I am going to try to enhance my plugin to see if I can get it to work. I
have some ideals that I am going to try. With the change in fastify to have
the original request as the raw property of the fastify request has made it
tricky, but I really think I can get it to work, maybe with using an
additional fastify hook.

Thanks for you question, and hopefully I can eventually answer yes to both
of the fastify issues.

Eric
